### PR TITLE
Add bcdedit trick for hypervisor troubleshooting

### PR DESCRIPTION
**Pull request by simonferquel**
### Proposed changes

In Docker Desktop for Windows troubleshooting, added information about enabling Hypervisor at Windows launch using bcdedit.

cc @mat007 @stephen-turner

### CODE REVIEW

File: docker-for-windows/troubleshoot.md [hunk 1]
```diff
@@ -206,11 +206,15 @@ Windows 10 Enterprise](https://github.com/docker/for-win/issues/74).
...
-If you have completed everything described above, and still have Docker Desktop faiing to start because of Hypervisor issues, this might be because the Hypervisor is installed but not launched at Windows Startup. Some tools (such as older versions of Virtual Box) and video game installers are known to do that.
-To make the Windows Hypervisor run at launch:
-- Open an administrative console prompt
-- Run `bcdedit /set hypervisorlaunchtype auto`
-- Restart Windows
+If you have completed the steps described above and are still experiencing
+Docker Desktop startup issues, this could be because the Hypervisor is installed,
+but not launched during Windows startup. Some tools (such as older versions of 
+Virtual Box) and video game installers require you to manually launch the Hypervisor 
+at Windows startup. To do this:
```
Comment (simonferquel) [inline docker-for-windows/troubleshoot.md:213]:
```suggestion
Virtual Box) and video game installers disable hypervisor on boot. To reenable it:
```

**Review by simonferquel — COMMENTED** [review]
Wording fix changed slightly the meaning of the paragraph

**Review by usha-mandya — APPROVED** [review]
LGTM

**Comment by simonferquel** [comment]
@stephen-turner can you PTAL ?

**Review by stephen-turner — APPROVED** [review]
LGTM

**Pull request merged**